ddtrace/tracer/spancontext.go

```diff
@@ -128,16 +127,30 @@ func (c *spanContext) baggageItem(key string) string {
 // finish marks this span as finished in the trace.
 func (c *spanContext) finish() { c.trace.finishedOne(c.span) }
 
+// samplingDecision is the decision to send a trace to the agent or not.
+type samplingDecision int64
+
+const (
+	// decisionNone is the default state of a trace.
+	// If no decision is made about the trace, the trace won't be sent to the agent.
```

Comment:
This is not true. None of the code paths make this happen.

Reply:
It's this code path here: https://github.com/DataDog/dd-trace-go/blob/4856aeaaf9b5ed02f7a19ee3a7c979dad2fa31ea/ddtrace/tracer/spancontext.go#L285

so decisionNone is not sent to the agent.